data_model/src/lib.rs

```diff
@@ -338,7 +343,81 @@ pub enum Value {
     /// [`struct@Hash`]
     Hash(Hash),
     /// Block
-    Block(BlockValue),
+    Block(BlockValueWrapper),
+}
+
+/// Cross-platform wrapper for `BlockValue`.
```

Comment:
IMO we introduce unnecessary complexity here by make platform specific implementation of `BlockValueWrapper`.

I think maybe just `Box<BlockValue>` could work?

Reply:
Please, read the requirements